alphadia/outputtransform.py

```diff
@@ -374,9 +376,44 @@ def build(
         _ = self.build_lfq_tables(folder_list, psm_df=psm_df, save=True)
         _ = self.build_library(base_spec_lib, psm_df=psm_df, save=True)
 
-        if self.config["transfer_learning"]["enabled"]:
+        if self.config["transfer_library"]["enabled"]:
             _ = self.build_transfer_library(folder_list, save=True)
 
+        if self.config["transfer_learning"]["enabled"]:
+            _ = self.build_transfer_model()
+
+    def build_transfer_model(self):
+        logger.progress("Train PeptDeep Models")
+
+        transfer_lib_path = os.path.join(
+            self.output_folder, f"{self.TRANSFER_OUTPUT}.hdf"
+        )
+        if not os.path.exists:
+            raise ValueError(
+                f"Transfer learning library was not found at {self.TRANSFER_OUTPUT}.hdf. Did you enable library generation?"
+            )
```

Comment:
I saw this pattern a log today already ..
Think about introducing one static method that does this check
```
def check_path_exists(path, "Transfer learning library was not found. Did you enable library generation?")
... # works for files and folders
```

Reply:
I've added an assert one-liner, what do you think?